index.js

```diff
@@ -50,7 +50,7 @@ try {
     TASK_COMMENT = core.getInput('task-comment'),
     PULL_REQUEST = github.context.payload.pull_request,
     REGEX = new RegExp(
-      `\\*\\*${TRIGGER_PHRASE}\\*\\* \\[(.*?)\\]\\(https:\\/\\/app.asana.com\\/(\\d+)\\/(?<project>\\d+)\\/(?<task>\\d+).*?\\)`,
+      `${TRIGGER_PHRASE} \\[(.*?)\\]\\(https:\\/\\/app.asana.com\\/(\\d+)\\/(?<project>\\d+)\\/(?<task>\\d+).*?\\)`,
```

Comment:
We can also make a space after `TRIGGER_PHRASE` optional or add it to trigger phrase to make it more flexible. Adding a sample of PR description will make it more clear for users, how to use it properly.

Reply:
https://github.com/actions/toolkit/issues/339

Reply:
I don't understand your comment, @iamalisajid . I see that there is an issue with leading spaces, but the way I understand this, it's not an issue here, correct?

Reply:
I also do agree, that you can simply update this REGEX to allow for zero or more spaces between the trigger phrase and the URL.

Reply:
Actually currently github have a issue logged that they remove leading spaces from the inputs as they are using `.trim()`. 
Due to which if the users event put any leading spaces in the trigger phrase it'll be ignored.

Reply:
Though, I've made the space optional using regex.